tests/treeTest.py

```diff
@@ -1,18 +1,15 @@
+import os
 import unittest
-
 from coalahtml.tree import Node, Tree
-from coalib.coala_main import run_coala
-from coalib.output.printers.ListLogPrinter import ListLogPrinter
-from coalahtml.helper import parse_file_dict
 
 
 class TreeTest(unittest.TestCase):
 
     def setUp(self):
-        log_printer = ListLogPrinter()
-        file_dict = run_coala(log_printer=log_printer,
-                              autoapply=False)[2]
-        self.FL = list(parse_file_dict(file_dict).keys())
+        self.FL = []
+        for root, _, file_list in os.walk(os.path.abspath('tests')):
+            for fname in file_list:
+                self.FL.append(os.path.join(root, fname))
```

Comment:
My points was to use `extend` instead of `apply`, as it's slightly faster. So you can do:
```python
self.FL.extend(os.path.join(root, fname) for fname in file_list)
```